package.json

```diff
@@ -55,6 +55,7 @@
     "ajv": "^5.5.2",
     "arangojs": "^5.8.0",
     "async": "^2.6.0",
+    "async-retry": "^1.2.1",
```

Comment:
this should trigger version bump at least, right?

Reply:
we will bump a version after a release, this does not trigger the version bump